src/App.tsx

```diff
@@ -1,34 +1,59 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions } from './features/todos';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const loadTodos = async () => {
+    setIsLoading(true);
+    try {
```

Comment:
You have to use `catch` block if you use `try`. You can handle errors inside it (on this stage you can do `console.warn` for example).